docs/releasenotes/2.3.1.rst

```diff
@@ -0,0 +1,12 @@
+2.3.1
+-----
+
+Security
+========
+
+Fix CVE-2014-1932, CVE-2014-1933
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. note:: More information about these vulnerabilities included in database records :cve:`CVE-2014-1932`, :cve:`CVE-2014-1933`
+
+Fix insecure use of tempfile.mktemp as reported in `Debian bug #737059 <https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=737059>`_.
```

Comment:
We can use intersphinx to link to the Python docs.

And let's wrap lines to around ~80 chars.

```suggestion
Fix insecure use of :py:func:`tempfile.mktemp` as reported in
`Debian bug #737059 <https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=737059>`_.
```